### ISSUE TITLE
Build theme in ESM project

### ISSUE THREAD

**Issue by QuentinDutot** [reshaped/community#79]
With `"type": "module"` in **package.json**, I'm getting an esm error when trying to build the theme from `reshaped theming --output src/themes`

I have to rename **reshaped.config.js** to **reshaped.config.cjs** but it's not supported: `Error: Reshaped config not found`

**Comment by blvdmitry** [comment]
Hey, thanks for sharing! Going to take a look this week.
Do you also have an error log of what happens when you try running the script without changing the file extension?

**Comment by QuentinDutot** [comment]
Sure, here it is:
```
reshaped-next\node_modules\.pnpm\file+reshaped-react-v1.7.0.tgz_j5lrhep7oqtn7q2g7uqfx7jqqa\node_modules\reshaped\bin\cli.js:14
        const config = require(path);
                ^

Error [ERR_REQUIRE_ESM]: require() of ES Module reshaped-next\reshaped.config.js from reshaped-next\node_modules\.pnpm\file+reshaped-react-v1.7.0.tgz_j5lrhep7oqtn7q2g7uqfx7jqqa\node_modules\reshaped\bin\cli.js not supported.
reshaped.config.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename reshaped.config.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in reshaped-next\package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

    at importJSConfig (reshaped-next\node_modules\.pnpm\file+reshaped-react-v1.7.0.tgz_j5lrhep7oqtn7q2g7uqfx7jqqa\node_modules\reshaped\bin\cli.js:14:17)
    at Command.<anonymous> (reshaped-next\node_modules\.pnpm\file+reshaped-react-v1.7.0.tgz_j5lrhep7oqtn7q2g7uqfx7jqqa\node_modules\reshaped\bin\cli.js:45:18)
    at Command.listener [as _actionHandler] (reshaped-next\node_modules\.pnpm\commander@8.0.0\node_modules\commander\lib\command.js:466:17)
    at reshaped-next\node_modules\.pnpm\commander@8.0.0\node_modules\commander\lib\command.js:1170:65
    at Command._chainOrCall (reshaped-next\node_modules\.pnpm\commander@8.0.0\node_modules\commander\lib\command.js:1088:12)
    at Command._parseCommand (reshaped-next\node_modules\.pnpm\commander@8.0.0\node_modules\commander\lib\command.js:1170:27)
    at Command._dispatchSubcommand (reshaped-next\node_modules\.pnpm\commander@8.0.0\node_modules\commander\lib\command.js:995:25)
    at Command._parseCommand (reshaped-next\node_modules\.pnpm\commander@8.0.0\node_modules\commander\lib\command.js:1136:19)
    at Command.parse (reshaped-next\node_modules\.pnpm\commander@8.0.0\node_modules\commander\lib\command.js:841:10)
    at Object.<anonymous> (reshaped-next\node_modules\.pnpm\file+reshaped-react-v1.7.0.tgz_j5lrhep7oqtn7q2g7uqfx7jqqa\node_modules\reshaped\bin\cli.js:64:9) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by blvdmitry** [comment]
Added .cjs support and included it into the 1.8 release: https://reshaped.so/content/changelog

Thanks again for the reported issue 🙏🏼
